app/models/spree/user_decorator.rb

```diff
@@ -18,9 +18,13 @@ def recent_order
     end
 
     def create_bronto_contact
-      contact = BrontoIntegration::Contact.new(bronto_token)
-      contact.find_or_create(email)
-      contact.update_status(email, 'active')
+      begin
+        contact = BrontoIntegration::Contact.new(bronto_token)
+        contact.find_or_create(email)
+        contact.update_status(email, 'active')
+      rescue Bronto::ValidationError
+        # noop
```

Comment:
@gbxl they were polluting the error reporting and they are one hundred percent redundant in this case. https://github.com/DynamoMTL/spree_bronto/commit/aef03db0b6cbb8c6362f27b54171fad127f29b69